### PR TITLE
Update style.css for Thai Font. Issue #59

### PR DESCRIPTION
**Pull request by imnok**
Update style.css for Thai Font. Issue #59

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -522,10 +514,24 @@ q:after {
 	text-rendering: auto;
 }
 
+/* Typography for Thai Font */
+
+html[lang="th"] h1, h2, h3, h4, h5, h6 {
```
Comment (josephfusco) [inline style.css]: Should all selectors here be specified more?

```
html[lang="th"] h1,
html[lang="th"] h2,
html[lang="th"] h3, 
html[lang="th"] h4, 
html[lang="th"] h5, 
html[lang="th"] h6 {
     letter-spacing: 0;
     line-height: 1.65;
}

html[lang="th"] body,
html[lang="th"] button,
html[lang="th"] input,
html[lang="th"] select,
html[lang="th"] textarea {
     line-height: 1.8;
}
```

Comment (imnok) [reply style.css]: @josephfusco You are right! All selectors should apply for Thai Font only. Can I fix and commit again?

Comment (grappler) [reply style.css]: @imnok Yes, you can edit the file and commit again. @melchoyce mentioned about not removing the spaces after the paragraphs. It is better to make such a change in a separate pull request if it is still needed.

Comment (imnok) [reply style.css]: @grappler Got it. Anyway, I removed only letter-spacing property of heading, so I'm going to recheck overall. Thank you for advice :)

**Comment by melchoyce** [comment]
Thanks @imnok!

**Comment by melchoyce** [comment]
@imnok — Spacing aside, how is the font? Are there any missing characters or anything that looks weird? Should we explore a specific Thai font stack?

**Comment by imnok** [comment]
Hi @melchoyce, I'm really sorry for my delay T T. Well, I just tested the theme with WordPress 4.7 Beta 4. It's very cool! 

To make it better, I think we can add some specific fonts for Thai as well. Here is my suggestion.

/* Typography for Thai Font */

html[lang="th"] h1,
html[lang="th"] h2,
html[lang="th"] h3,
html[lang="th"] h4,
html[lang="th"] h5,
html[lang="th"] h6 {
	line-height: 1.65;
	font-family: "Sukhumvit Set", "Helvetica Neue", helvetica, arial, sans-serif;
}

html[lang="th"] body,
html[lang="th"] button,
html[lang="th"] input,
html[lang="th"] select,
html[lang="th"] textarea {
	line-height: 1.8;
	font-family: "Helvetica Neue", helvetica, arial, sans-serif;
}

**Comment by melchoyce** [comment]
Hi @imnok! Thanks for this. Can you create a trac ticket? Twenty Seventeen has been merged into WordPress core, so we're finishing up development there: https://core.trac.wordpress.org/newticket

Please begin the ticket title with "Twenty Seventeen: " and select the "Bundled Themes" component.

Thanks!

**Comment by imnok** [comment]
Thank you @melchoyce :) You can check my ticket here https://core.trac.wordpress.org/ticket/38937

**Comment by melchoyce** [comment]
Thanks @imnok!

**Pull request merged**